challenges/Front-End Challenge/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import './App.css';
+import { Navbar, Footer, Form } from './components/index';
+import { loadReCaptcha } from 'react-recaptcha-v3';
+
+function App() {
+
+  useEffect(() => {
+    //This is to load the Google ReCaptcha with the sitekey
+    loadReCaptcha('6Le3WMAUAAAAAOCR9AMk7lEj-N8S_bo-nLfRKOXW');
+  })
```

Comment:
I don't have too much experience with the Google ReCaptcha package, but I believe it could work if I moved this functionality to a backend server and have it load in once the server starts.

Reply:
Another way to keep track of these keys is to have a separate file that you import the values from. That file could be moved to the backend and simply import the value as a variable name to keep the value of it secret.

Reply:
This is a great article around `useEffect` that leads to where I was asking with this question. https://overreacted.io/a-complete-guide-to-useeffect/

Basically, you'd change the code to look like this:
```
useEffect(() => {
    //This is to load the Google ReCaptcha with the sitekey
    loadReCaptcha('6Le3WMAUAAAAAOCR9AMk7lEj-N8S_bo-nLfRKOXW');
  }, [])
```
That extra array at the end tells the effect to only trigger when some dependency of the effect has changed. Because in this case we have no dependencies, aside from the loadRecaptcha function which does not count in this context, this effect will only run once, thus only running loadReCaptcha once.

Reply:
Oh okay thanks! Did not know that, definitely useful for future projects though.